Assets/__Scripts/UI/Themes/ThemeSO.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using TMPro;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "NewTheme", menuName = "ScriptableObjects/Theme", order = 1)]
+public class ThemeSO : ScriptableObject
+{
+    public Color backgroundPrimary;
```

Comment:
Would it hurt much to switch these to `PascalCase` to fit .NET Naming Guidelines? Same question also applies to other classes in the PR.